### ISSUE TITLE
Measuring Tool

### ISSUE THREAD

**Issue by MrVirite** [mapeditor/tiled#3067]
Simple tool to allow measuring the distance between two points. 
Could output both pixel distance and tile distance.

**Comment by bjorn** [comment]
It's a useful suggestion, though I'm not sure where to put this on the priority scale at the moment.

If you need this kind of functionality right now, I'd suggest to write a custom [scripted tool](https://www.mapeditor.org/docs/scripting/modules/tiled.html#registertool) that outputs this information. Unfortunately scripted tools currently can't show arbitrary graphics or text in the map view, but its output could go to the Console for now.

**Comment by eishiya** [comment]
> but its output could go to the Console for now.
If it's a scripted Tool, the output could go in the Status Bar as well, where I think it would be quite intuitive (that's usually where this info goes in other tools, too!).

**Comment by bjorn** [comment]
Right, a scripted tool can set its [`statusInfo`](https://www.mapeditor.org/docs/scripting/interfaces/tool.html#statusinfo) property in `updateStatusInfo`. :+1:

**Comment by firstmiddlelast** [comment]
I did a pathing / surface measurement tool in my [tiled extensions repository](https://github.com/firstmiddlelast/tiled-extensions), that does just that. It does not output pixel distance, though I could add that feature if someone finds it useful.
